interactions/models/command.py

```diff
@@ -110,7 +110,9 @@ def __init__(self, **kwargs) -> None:
         if self._json.get("options"):
             self._json["options"] = [option._json for option in self.options]
         if self._json.get("choices"):
-            self._json["choices"] = [choice._json for choice in self.choices]
+            self._json["choices"] = [
```

Comment:
```
            if isinstance(self._json.get("choices"), dict):
                self._json["choices"] = [choice for choice in self.choices]
            else:
                self._json["choices"] = [choice._json for chocie in self.choices]
```

We can build a double check instead of a ternary operation that you have here.